features/canvas.yml

```
@@ -1,15 +1,16 @@
-draft_date: 2024-06-05
-name: <canvas>
-description: The `<canvas>` element TODO.
+name: Canvas API
+description: The canvas API provides a means for drawing graphics via JavaScript and the HTML `<canvas>` element.
```

Should this feature represent just the HTML `<canvas>` element, or also 2D canvas?

I think we should add features for 2D canvas and WebGPU and put them on equal footing, even though 2D canvas is much older and shipped together with `<canvas>`. (Developers shouldn't need to know this in 2024.)

That would leave `<canvas>` as a feature that does nothing by itself, just a way to access other features. I think that's OK, but the name and description would be a bit different. @ddbeck thoughts?

---

Yes, I agree with this. The name here would return to `<canvas>` and the description might look something like this, where we gesture in the direction of other features you would use with this one:

```suggestion
name: Canvas API
description: The `<canvas>` HTML element represents a space to draw graphics in, using the 2D canvas API or 3D WebGPU API.
```